tsconfig.json

```diff
@@ -89,15 +89,5 @@
     "entryDocument": "index.md",
   },
   "include": ["src/**/*.ts", ".eslintrc.js"],
-  "exclude": [
-    "**/node_modules",
-    "tests/**/*.js",
-    // These end-to-end tests reference code in `.codesandbox`,
-    // which is not part of this project and should not be compiled together with it.
-    // Not excluding this will lead to the root of the repository being used
-    // as the root directory for compilation (instead of /src),
-    // meaning that files that would otherwise be included in /dist directly
-    // (e.g. dist/index.d.ts) will then be built to /dist/src.
-    "e2e",
```

Comment:
I did verify that the build output remained consistent after this change, though someone may wish to double check.